### PR TITLE
use dlog to print debug information

### PR DESCRIPTION
**Pull request by felix5572**
use dlog to print debug information

### CODE REVIEW

**Review by njzjz — CHANGES_REQUESTED** [review]
Can you look at the output of the GitHub Actions? There are still some outputs produced by `print`.

**Comment by njzjz** [comment]
https://github.com/deepmodeling/dpdispatcher/pull/53/checks?check_run_id=2804850634#step:5:67

**Comment by felix5572** [comment]
And if we cloud recover support for python==3.6 ?  I realize that there are many python3.6 here. and  dpdispatcher can not be installed if python_requires="~=3.7".

File: dpdispatcher/dp_cloud_server.py [hunk 1]
```diff
@@ -64,14 +58,14 @@ def do_submit(self, job):
     def check_status(self, job):
         if job.job_id == '':
             return JobStatus.unsubmitted
-        print('debug: check_status', job)
+        dlog.debug('debug: check_status', job)
```
Comment (njzjz) [inline dpdispatcher/dp_cloud_server.py:61]: I think dlog doesn't support multiple args. Please check it.

File: dpdispatcher/base_context.py [hunk 2]
```diff
@@ -19,7 +19,7 @@ def load_from_dict(cls, context_dict):
         try:
             context_class = cls.subclasses_dict[context_type]
         except KeyError as e:
-            print('debug:context_type;subclasses_dict', context_type, cls.subclasses_dict)
+            print('KeyError:context_type;subclasses_dict', context_type, cls.subclasses_dict)
```
Comment (njzjz) [inline dpdispatcher/base_context.py:22]: You may try `dlog.error` or `dlog.exception`.

**Comment by felix5572** [comment]
fixed @njzjz  dlog

**Comment by felix5572** [comment]
@njzjz  and fixed. query database in submitting jobs. dpdispatcher will now query database after submitting all the jobs
image

File: dpdispatcher/base_context.py [hunk 3]
```diff
@@ -19,7 +19,7 @@ def load_from_dict(cls, context_dict):
         try:
             context_class = cls.subclasses_dict[context_type]
         except KeyError as e:
-            print('debug:context_type;subclasses_dict', context_type, cls.subclasses_dict)
+            dlog.error("KeyError:context_type; context_type:{context_type}; cls.subclasses_dict:{cls.subclasses_dict}")
```
Comment (njzjz) [inline dpdispatcher/base_context.py:22]:
```suggestion
            dlog.error(f"KeyError:context_type; context_type:{context_type}; cls.subclasses_dict:{cls.subclasses_dict}")
```

File: dpdispatcher/machine.py [hunk 4]
```diff
@@ -96,7 +96,7 @@ def load_from_dict(cls, machine_dict):
         try:
             machine_class = cls.subclasses_dict[batch_type]
         except KeyError as e:
-            print('debug:batch_type;subclasses_dict', batch_type, cls.subclasses_dict)
+            dlong.error("KeyError:batch_type; cls.subclasses_dict:{cls.subclasses_dict}; batch_type:{batch_type};")
```
Comment (njzjz) [inline dpdispatcher/machine.py:99]:
```suggestion
            dlong.error(f"KeyError:batch_type; cls.subclasses_dict:{cls.subclasses_dict}; batch_type:{batch_type};")
```

**Review by njzjz — APPROVED** [review]
I will approve after fixing these two

**Review by njzjz — APPROVED** [review]

**Pull request merged**